rbc/omniscidb.py

```diff
@@ -928,7 +933,7 @@ def _make_udtf(self, caller, orig_sig, sig):
             name + sig.mangling(),
             sizer_type, sizer_index,
             inputArgTypes, outputArgTypes, sqlArgTypes,
-            annotations)
+            annotations, uses_manager)
```

Comment:
Instead of uses_manager, use function annotations. That is, omniscidb treats the last annotation in annotations list as function annotation. So, insert somewhere above the following code:
```python
function_annotation = dict()  # TODO: retrieve function annotations from orig_sig
if uses_manager:
    function_annotations['uses_manager'] = True
annotations.append(function_annotation)
```

Reply:
Just pushed changes to implement this. See the omnisci PR as well: [PR 6035](https://github.com/omnisci/omniscidb-internal/pull/6035). I tried to implement it in a way that keeps the interfaces for rbc and compile-time UDTFs as close as possible.